publish/publish-blob.proj

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <UsingTask TaskName="DownloadFromAzure" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.CloudTestTasks.dll" />
+
+  <Import Project="$(PackagesDir)/$(FeedTasksPackage.ToLower())/$(FeedTasksPackageVersion)/build/$(FeedTasksPackage).targets" />
+
+  <Target Name="Build"
+          DependsOnTargets="ValidateProperties;
+                            DownloadFilesFromContainer;
+                            SignSymbolPackages;
+                            PublishPackagesToBlobFeed;
+                            PublishFilesToBlobFeed;
+                            UpdateVersionsRepo" />
+
+  <Target Name="ValidateProperties">
+    <Error Condition="'$(AzureAccessToken)' == ''" Text="Missing required property 'AzureAccessToken'" />
+    <Error Condition="'$(AzureAccountName)' == ''" Text="Missing required property 'AzureAccountName'" />
+    <Error Condition="'$(ContainerName)' == ''" Text="Missing required property 'ContainerName'" />
+    <Message Condition="'$(WindowsSdkDir)' == ''" Text="Windows SDK not found. Symbols packages will not be signed." />
+    <Error Condition="'$(PublishBlobFeedUrl)' == ''" Text="Missing required property 'PublishBlobFeedUrl'" />
+    <Error Condition="'$(PublishBlobFeedKey)' == ''" Text="Missing required property 'PublishBlobFeedKey'" />
+
+    <PropertyGroup>
+      <!-- map the properties to the values expexed by the Feeds.targets file -->
+      <ExpectedFeedUrl>$(PublishBlobFeedUrl)</ExpectedFeedUrl>
+      <AccountKey>$(PublishBlobFeedKey)</AccountKey>
+    </PropertyGroup>
+  </Target>
+
+  <Target Name="DownloadFilesFromContainer">
+    <!-- TODO: We should also think about pulling down the checksums or recomputing them -->
```

Comment:
I've decided for now to ignore the checksums but we will eventually need to either re-compute or download them to publish as well.